atramhasis/scripts/migrate_sqlalchemy_providers.py

```diff
@@ -187,8 +187,9 @@ def migrate(skos_registry: Registry, session: Session):
             )
         else:
             provider.metadata['atramhasis.id_generation_strategy'] = 'NUMERIC'
-        db_provider.meta = json.loads(json.dumps(provider.metadata, default=json_serial))
-
+        meta = json.loads(json.dumps(provider.metadata, default=json_serial))
+        meta["id"] = provider.conceptscheme_id
```

Comment:
This makes the problem worse as it now fully overwrites the old provider id with the conceptscheme_id. We want the old provider ID to remain functional.